extension/plg_sys_webwinkelkeur/WebwinkelKeurHikaShopPlatform.php

```diff
@@ -96,7 +96,9 @@ public function getOrderCustomerName($order) {
         return $order['customer_name'];
     }
 
-    public function getOrderLanguage($order) {}
+    public function getOrderLanguage($order) {
+        return isset($this->getOrderData($order)['order']['order_lang']) ? $this->getOrderData($order)['order']['order_lang'] : 'cus';
```

Comment:
I would have used "null coalescing operator" but the extension is in PHP 5.5

Reply:
Why `cus`?

Reply:
I remember in another plugin we had it 'cus' when it was not assigned, decided to make it the same.